scheduler/scheduler_dfs.go

```diff
@@ -198,10 +199,19 @@ func (s *syncClient) resolveResourcesDfs(ctx context.Context, table *schema.Tabl
 				wg.Wait()
 				return
 			}
+			tableSemVal, _ := s.scheduler.singleTableConcurrency.LoadOrStore(table.Name+"-"+client.ID(), semaphore.NewWeighted(int64(s.scheduler.singleTableMaxConcurrency)))
```

Comment:
It looks to me that we'll create a semaphore for each client/table pair, effectively doing nothing.

Shouldn't we instead use either only the table name or `s.scheduler.tableSems[depth]`?